lib/pollTravis.js

```diff
@@ -129,7 +63,9 @@ function pollTravisBuildBySha (options, checkNumber) {
       return console.error(`! ${prInfo} Got error when retrieving Travis builds`, err.stack)
     }
 
-    const matchingCommit = res.commits.find((commit) => commit.sha === shaToMatch)
+    const matchingCommit = res.commits.find((commit) => {
+      return commit.sha === shaToMatch || commit.pull_request_number === prToMatch
```

Comment:
Hmm, I think we somehow need to ensure we prioritize matching the last commit in a pr to avoid races.